tools/tpm2_getcap.c

```diff
@@ -834,7 +833,7 @@ tool_rc tpm2_tool_onrun(ESYS_CONTEXT *context, tpm2_option_flags flags) {
         return tool_rc_success;
     }
 
-    /* List a capability, ie -c <arg> option */
+    /* List a capability, ie <arg> option */
```

Comment:
This is a bit misleading since `<arg>` can be anything. I'd rather write `<capability group>`.